### DISCUSSION TITLE
I need a CPA

### DISCUSSION THREAD

**Discussion by vittorioviarengo** [hickeng/financial#153] [Finding a CPA]
Anybody found a good CPA that has a good handle on all of this? Thanks in advance for all the amazing work you shared here!

**Comment by hickeng** [comment]
I've got a couple of referrals from people, but they're all too busy to engage in new and complicated accounts until after the tax deadline.
I'm filing an extension then reaching out to those referrals after April 15th.

If people have recommendations/referrals please to note them!

**Reply by 1Britt** [reply]
I am an accountant and  am very familiar with VMWare/Broadcom situation.   My website is: www.1britt.com

**Comment by vittorioviarengo** [comment]
Thank you as always.

**Comment by vittorioviarengo** [comment]
Thanks Britt, let me see if I can file an extension, then engage with you
on this. Thanks

**Reply by 1Britt** [reply]
Sounds great! I will be on the lookout for an email from you. 

Hope you have a great day!

**Comment by MonadicT** [comment]
I filed for an extension and now I am looking for a CPA. Preferably someone in the Bay Area. Any recommendations?

Thanks!

**Comment by ryanconley1** [comment]
http://simcpa.com/ out of Los Angeles worked with several current/former VMware employees through this transition.

**Comment by pps8888** [comment]
I need a CPA  as well -- any recommendations from above or new ones?  Looking for someone who may have already done this for other employees and so may know the drill easier. thanks

**Comment by MonadicT** [comment]
I had my taxes prepared by Brandon at https://amaralfp.com/. As a first time user of a tax preparer, the process went smoothly and price was fair. His services also included audit defense and I am very happy with the outcome.

I don’t know if you are looking for a CPA or just tax prep and also not sure if Brandon is a CPA or not. I recommend him without any reservations if you are looking for help filing taxes.

No relationship, satisfied customer…

**Comment by pps8888** [comment]
THANKS very appreciative -- how did he do with the complete mess of
vmware/avgo shares and determining the right cost basis etc?
Also how much did he charge?

**Comment by clvmware** [comment]
Anybody who has a CPA recommendation in the bay area that can help me? I've downloaded all the doc but having issues with entering info in the spreadsheet and would love some help.

**Reply by hickeng** [reply]
Contact me on either the alumni slack or google chat and I can help you with the sheet. Can't help with the CPA.

**Reply by phene** [reply]
Michael Nguyen at Johanson & Yau accounting corporation is in the bay area and has experience with this exact transaction.

[michaeln@jyac.com](mailto:michaeln@jyac.com)

**Comment by 1Britt** [comment]
I have worked with a lot of wonderful VMWare employees on this and would be more than happy to help! Www.1britt.comThanks,Brittani PurnellOn Sep 11, 2024, at 3:28 PM, Geoffrey Hichborn ***@***.***> wrote:﻿
Michael Nguyen at Johanson & Yau accounting corporation has experience with this exact transaction.
***@***.***
